### PR TITLE
Watch performance

### PR DESCRIPTION
**Pull request by Chibikuri**
# What I did
Wrote a document to verify the simulation results and tutorials for performance analysis. 
Related Issue: #142

### CODE REVIEW

File: quisp/modules/BellStateAnalyzer.cc [hunk 1]
```diff
@@ -29,8 +30,14 @@ class BellStateAnalyzer : public cSimpleModule {
  private:
   // for performance analysis
   int n_res = 0;
+  int trials = 0;
   simsignal_t GOD_num_resSignal;
-
+  std::string BSA_perf_output_filename;
+  std::vector<simtime_t> creation_time;
+  simsignal_t Average_Num_TrialSignal;
```
Comment (zigen) [inline quisp/modules/BellStateAnalyzer.cc:37]: could you use lower_case name for the local variables in your change? 
for example, this should be `average_num_trial_signal`.

see also https://google.github.io/styleguide/cppguide.html#General_Naming_Rules

File: quisp/modules/BellStateAnalyzer.cc [hunk 2]
```diff
@@ -332,6 +339,25 @@ void BellStateAnalyzer::finish() {
   std::cout << "Success = " << DEBUG_success << "\n";
   std::cout << "darkcount_count_left = " << DEBUG_darkcount_left << ", darkcount_count_right =" << DEBUG_darkcount_right << ", darkcount_count_both = " << DEBUG_darkcount_both
             << "\n";
+  std::cout << "total BSA performance"
+            << "\n";
+  // filename for recoding bsa performance
+  // std::string file_name = BSA_perf_output_filename;
+  // int file_size = file_name.size();
+  // create file
+  std::string a = std::to_string(duration);
```
Comment (zigen) [inline quisp/modules/BellStateAnalyzer.cc:348]: please don't give a meaningless name such as `std::string a` for readability.
in this case, we can conbine L348 and L349 like this
```
   std::string file_name = "num_trials" + std::to_string(duration); 
```

**Review by zigen — CHANGES_REQUESTED** [review]

File: quisp/modules/BellStateAnalyzer.cc (hunk 1 above)
Comment (Chibikuri) [reply quisp/modules/BellStateAnalyzer.cc:37]: OK, I fixed it. Thank you.

File: quisp/modules/BellStateAnalyzer.cc (hunk 2 above)
Comment (Chibikuri) [reply quisp/modules/BellStateAnalyzer.cc:348]: I got it. Thank you for notifying.

**Review by rdviii — APPROVED** [review]

**Pull request merged**